src/App.tsx

```diff
@@ -11,11 +11,19 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((prevState) => {
+      return {
+        count: prevState.count + 1,
+      };
+    });
   };
 
   add100 = () => {
-    // write code here
+    this.setState((prevState) => {
+      return {
+        count: prevState.count + 100,
+      };
+    });
```

Comment:
```suggestion
    this.setState((prevState) => ({
      count: prevState.count + 100,
    }));
```
checkList: try to solve the task in 6 lines of code :)

Reply:
Thanks, i forogot for this)